### PR TITLE
Always go to Stream (group home) when navigating to a new group or context

### PR DESCRIPTION
**Pull request by tibetsprague**

### CODE REVIEW

**Comment by tibetsprague** [comment]
fixes https://github.com/Hylozoic/hylo-evo/issues/1289

**Comment by lorenjohnson** [comment]
Curious more then concerned: Does this still maintain Map context when switching views?

File: src/util/navigation.js [hunk 1]
```diff
@@ -51,21 +51,6 @@ export function baseUrl ({
   }
 }
 
-export function contextSwitchingUrl (newParams, routeParams) {
```
Comment (thomasgwatson) [inline src/util/navigation.js:54]: yaya deleting code!

**Review by thomasgwatson — APPROVED** [review]
lgtm

**Comment by tibetsprague** [comment]
decided not to do the work to have the map stay on the map. i think lets try it this way and see if we get feedback from folks about it....

**Pull request merged**